Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.13 AS builder
+
+COPY . insights-results-aggregator
+
+RUN cd insights-results-aggregator && \
+    make build
+
+FROM registry.access.redhat.com/ubi8-minimal
+
+COPY --from=builder /go/insights-results-aggregator/insights-results-aggregator .
+
+RUN chmod a+x /insights-results-aggregator
+
+USER 1001
+
+CMD ["/insights-results-aggregator"]
```

Comment:
You should define mode ("consumer" or "server") in arguments, could it be as ENV var?

Reply:
it will be 'fixed' in https://github.com/RedHatInsights/insights-results-aggregator/pull/37

Reply:
@tisnik thank you!